packages/nextjs/hooks/scaffold-eth/useDeployedContractNames.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { useProvider } from "wagmi";
+import { getTargetNetwork } from "~~/utils/scaffold-eth";
+
+import contracts from "~~/generated/hardhat_contracts.json";
```

Comment:
Check `useDeployedContractInfo.ts` on how we are handling the import of this file.

I think we should avoid the NextJS "full-modal" error when people forget to `yarn deploy`. It happens on every page, not only on debug.

![image](https://user-images.githubusercontent.com/2486142/220956274-afa2487d-941d-4de9-b3a9-b9759e140382.png)

In some cases, people would also use SE2 and won't need a contract (like doing wallet signing to a backend).

It'll also avoid the github action error (although we might merge this at some point https://github.com/scaffold-eth/se-2/pull/191)

Reply:
Ah yes, good point :+1: